rpcs3/Emu/RSX/VK/vkutils/device.cpp

```diff
@@ -210,6 +210,11 @@ namespace vk
 #endif
 			}
 
+			if (gpu_name.find("Apple") != umax)
```

Comment:
You should also add the driver ID in the switch below.

Reply:
Let's not forget the 5+ other switches

Reply:
You can only hit the if case if driverId is 0 (or maybe moltenvk does not expose the KHR_maintenance2 extension?)
I asked to add it because it is correct. Moltenvk is not a finished product. What happens if it goes down the driverID path in future? Not adding it in both paths is just sloppy. Alternatively you can hoist the name check outside the if..else block if you really must, but we cannot leave logical holes in the code like this.

Reply:
Good point.
I'll move it outside the if/else block for now, and maybe move it back later if moltenVK's implementation changes.

Reply:
We still have a bunch of other switches that query driver_vendor. Just mentioning it again.

Reply:
I'll also handle those.
It'll need more testing to find the optimal path for Apple GPUs, though.